spec/unit/php/parsers/composer_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+RSpec.describe Spandx::Php::Parsers::Composer do
+  subject { described_class.new(catalogue: catalogue) }
+
+  let(:catalogue) { Spandx::Spdx::Catalogue.from_file(fixture_file('spdx/json/licenses.json')) }
+
+  describe '#parse small lock file' do
+    let(:lockfile) { fixture_file('composer/composer.lock') }
+    let(:result) { subject.parse(lockfile) }
+
+    expected_array = ['phpdocumentor/reflection-common@2.0.0',
+                      'phpdocumentor/type-resolver@1.0.1',
+                      'symfony/polyfill-ctype@v1.14.0',
+                      'webmozart/assert@1.7.0',
+                      'doctrine/instantiator@1.3.0',
+                      'hamcrest/hamcrest-php@v2.0.0',
+                      'mockery/mockery@1.3.1']
```

Comment:
I'm surprised this works. Should the `expected_array` be wrapped in a let?

.i.e.

```ruby
let(:expected_array) { [..] }
```